src/features/auth/forgottenPassword/ForgottenPassword/ForgottenPassword.tsx

```diff
@@ -172,6 +117,136 @@ export const ForgottenPassword: FunctionComponent = () => {
   )
 }
 
+const useForgottenPasswordForm = (useFormMethods: Omit<UseFormReturn<FormValues>, 'control'>) => {
+  const { navigate } = useNavigation<UseNavigationType>()
+  const {
+    setError,
+    clearErrors,
+    setValue,
+    watch,
+    formState: { errors },
+  } = useFormMethods
+  const { email, isFetching, isDoingReCaptchaChallenge } = watch()
+
+  // Little helper method to make it easier to set error
+  const setCustomError = useCallback(
+    (field: FieldPath<FormValues>, message: string) => {
+      return setError(field, { type: 'custom', message })
+    },
+    [setError]
+  )
+
+  const networkInfo = useNetInfoContext({
+    onNetworkRecovered() {
```

Comment:
On a plusieurs type de perte de réseau:

`isInternetReachable` et `isConnected`, est-ce que tu peux utiliser ces nominations qu'on sache ce qu'on manipule sans avoir a apprendre des renommages qui rende le code moins lisible ?  Tu peux juste mettre un `Not` après le `is`

Reply:
J'ai pas tout compris. Autant je suis OK sur le renommage de `onNetworkLost` en `onConnectionLost` par exemple, et à l'avenir ajouter un `onInternetLost` pour le `isInternetReachable` mais pour la partie 
> Tu peux juste mettre un `Not` après le `is`

Je sèche :P

Reply:
Je comprends, je propose :   

`isConnected` : `onConnectionLost` et `onConnection`
`isInternetReachable` : `onInternetConnectionLost` et `onInternetConnection`